sdk/core/core-http/Changelog.md

```diff
@@ -1,216 +0,0 @@
-# Changelog
-
-## 1.8.10 - 2019-06-05
```

Comment:
Did you intend to also remove the changelog.md?

Reply:
Ah no ... good catch. I'll fix that up.